webpack.config.js

```diff
@@ -1,43 +1,18 @@
+/* eslint-env node */
 const path = require('path');
-const pkg = require('./package.json');
-const camelcase = require('camelcase');
-const process = require('process');
-const webpack = require('webpack');
-const env = process.env;
-const NODE_ENV = env.NODE_ENV;
-const MIN = env.MIN;
-const PROD = NODE_ENV === 'production';
 
-let config = {
-  devtool: PROD ? false : 'inline-source-map',
+module.exports = {
   entry: './src/index.js',
+  devServer: {
+    publicPath: '/dist/',
+  },
   output: {
-    path: path.join( __dirname ),
-    filename: pkg.name + '.js',
-    library: camelcase( pkg.name ),
-    libraryTarget: 'umd'
+    path: path.join(__dirname, 'dist'),
+    filename: 'cytoscape-elk.js',
+    library: 'cytoscapeElk',
+    libraryTarget: 'umd',
   },
   module: {
-    rules: [
-      { test: /\.js$/, exclude: /node_modules/, use: 'babel-loader' }
-    ]
+    rules: [{ test: /\.js$/, exclude: /node_modules/, use: 'babel-loader' }],
   },
-  externals: PROD ? {
```

Comment:
The externals are important for keeping the elk dependency separate and for making sure that the different variable names work in different environments, e.g. globals under `window` versus `import`.

Reply:
But they won't be needed when bundling Elk, as this PR does.

Reply:
The consumer should have the ability to upgrade Elk to compatible versions without upgrading the extension itself.

Reply:
Reverted.